tests/morale_test.cpp

```diff
@@ -192,6 +192,95 @@ TEST_CASE( "player_morale" )
         }
     }
 
+    GIVEN( "moderate masochistic personality disorder" ) {
```

Comment:
"personality disorder" - I'd drop the "medical" language here and just call it a "trait", like it is in game.